ambari-admin/src/main/resources/ui/ambari-admin/package.json

```diff
@@ -5,50 +5,53 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
-    "build": "tsc -b && vite build",
-    "lint": "eslint .",
+    "build": "tsc && vite build",
```

Comment:
Using `tsc` for individual .ts files isn't ideal. The `tsc -b` command in package.json is a better choice as it helps identify dependencies [across files](https://www.typescriptlang.org/docs/handbook/compiler-options.html#compiler-options), leading to a more robust application. It's important not to "hide" issues, as that defeats the purpose of TypeScript checks.

There's a clear issue in `ambari-admin/src/main/resources/ui/ambari-admin/src/screens/Views/types.ts` : 

<img width="963" alt="image" src="https://github.com/user-attachments/assets/14fe4181-c0ab-4ef9-97b6-c1fc6fa85d24">


The file enums.ts doesn't exist, which will definitely cause the build to fail. Also, please have @himanshumaurya09876  take a closer look to fix this build error.